### DISCUSSION TITLE
In dafoam tutorial SnappyHexmesh

### DISCUSSION THREAD

**Discussion by ryupuriwat** [mdolab/dafoam#635] [General]
Now I am working on snappyHexmesh and I find the problem that when I try to make the first layer thickness to become thinner the first layer is not generate. I set the first layer thickness as below 
![ee2e644fc020fca6b4d0d8fd553f3ee](https://github.com/mdolab/dafoam/assets/109540234/d9149036-f7e4-427b-9a05-bfc4f6b99d87)
because in my case CT rotor blade Mtip is at 0.51 so when I use first layer calculator with y+=30 as profressor told other DAFoam user that this value is good. I get the first layer height as below  
![abebf6b3ceace4e77c02ea934c07ec6](https://github.com/mdolab/dafoam/assets/109540234/97d5aab7-86e6-4f74-9734-9ea9733dbea8)  I have notice that almost every DAFoam tutorial that use snappyhexmesh use to set minThickness as 0.001 and Final layer thickness as 0.4, could you please tell me why original tutorial use this value eventhough M=0.85 as shown in the figure below 
![0a9e56b45fc729996f4bda8968095ad](https://github.com/mdolab/dafoam/assets/109540234/bf4b1fff-081a-46fb-a8fe-336dbb0c5ebf) could you please give me suggestion If I would like to make the mesh that first layer height is 1.4e-4 works which parameter should I modify or for snappyHexmesh I can not make first layer thickness to this value and why the tutorial use the first layer thickness which is a bit thick?

**Comment by friedenhe** [comment]
Unfortunately, the behavior you saw is a known issue in snappyHexMesh. That is why we do not use snappyHexMesh anymore these days. Please try Pointwise or Ansys Meshing. They can generate high quality prism mesh for the boundary layer.
